packages/runtime/src/execute/expression.ts

```diff
@@ -146,11 +146,16 @@ export const wrapOperation = (
 
         let firstFrame;
 
-        // find the first error from a file or the VM
-        // (this cuts out low level language errors and stuff)
+        // find the first frame from an adaptor or the VM
+        // (this cuts out low level language errors like TypeError)
         do {
           const next = frames.shift();
-          if (/(@openfn\/language-)|(vm:module)/.test(next)) {
+          if (
+            // detect an adaptor prod, adaptor monorepo, or vm frame
+            /(@openfn\/language-)|(adaptors\/packages\/.+\/dist)|(vm:module)/.test(
```

Comment:
This is a bit hairy

We're looking in the callstack or a path like `adaptors/packages/common/dist`

Because the path to an adaptor in the monorepo isn't terribly distinctive.  path like`packages/*/dist` feels like it might erroneously match against many paths (then again maybe not? Most node_modules code would be like `packagename/dist`, with nothing in between.

Anyway, because of this, I wanted to put the repo name in the path.

But that doesn't work for anyone who forks the repo with a different name locally. In those cases, the path won't match and bad errors will be triggered, which leads to  misleading error reports when developing adaptors.

Perhaps I can do a regex like "not node_modules and packages/./dist"

Reply:
Ah it's fine - I'm sure `packages/./dist` is all we need for this. There's only a small amount of non-vm error cases that we need to handle. Let's try it out.